Components/Button/button.css

```diff
@@ -27,6 +28,7 @@
         background-color: var( --accent-color);
         color: var( --secondary-white);
         border: 2px solid transparent;
+        cursor: pointer;
```

Comment:
you could have added coursor:pointer in btn class above instead of adding it in every other button class

Reply:
Thanks for the suggestion. Will make the change